.editorconfig

```diff
@@ -0,0 +1,14 @@
+# http://editorconfig.org
```

Comment:
Not sure if we still need this file now we have Prettier.

Reply:
Prettier only supports certain files / file types, right?